lighthouse-extension/app/src/popup.js

```diff
@@ -91,34 +77,42 @@ document.addEventListener('DOMContentLoaded', _ => {
     return listItem;
   };
 
-  const generateOptionsList = list => {
+  const generateOptionsList = (list, selectedAudits) => {
     const frag = document.createDocumentFragment();
 
     aggregations.forEach(aggregation => {
-      frag.appendChild(createOptionItem(aggregation.name));
+      frag.appendChild(createOptionItem(aggregation.name, selectedAudits.indexOf(aggregation.name) > -1));
```

Comment:
oh wow. did not expect that. 

![image](https://cloud.githubusercontent.com/assets/39191/17728527/7a7f592e-6414-11e6-9dd2-1b281778302c.png)

bummer.